src/android/notification/Notification.java

```diff
@@ -175,6 +175,10 @@ public void schedule() {
                 context, 0, intent, PendingIntent.FLAG_CANCEL_CURRENT);
 
         if (isRepeating()) {
+            if (wasInThePast()) {
+                triggerTime = System.currentTimeMillis();
```

Comment:
Yeah, we only want to override the `triggerTime` if it `wasInThePast()`.

This is to mimic the former functionality of `getTriggerTime()`